nemo_curator/classifiers/domain.py

```diff
@@ -37,40 +35,14 @@ class DomainModelConfig:
     max_len = 512
```

Comment:
Minor non blocking NIT

@ryantwolf / @sarahyurick , Are we sure the `max_len` for domain is `512` and `quality` is `1024`. Might be worth double checking this.

Reply:
Double checked, this is correct.